API/index.js

```diff
@@ -0,0 +1,53 @@
+let input = document.querySelector("input");
+let imgContainer = document.querySelector(".photo")
+let inputValue;
+
+function imgCreate(data) {
+  for (let i = 0; i < data.results.length; i++) {
+    let img = document.createElement("img");
+    img.className = "picture";
+    img.src = data.results[i].links.download;
+    imgContainer.append(img);
+  }
+}
+
+input.addEventListener("keydown", (event) => {
+
+  if (event.key === "Enter") {
+    document.querySelector(".photo").textContent = "";
+
+    inputValue = input.value;
+
+    let promise = fetch('https://api.unsplash.com/search/photos?query=' + input.value + '&per_page=30&client_id=SouHY7Uul-OxoMl3LL3c0NkxUtjIrKwf3tsGk1JaiVo')
+
+      .then(response => {
+        return response.json();
+      })
+
+      .then(data => {
+        imgCreate(data);
+      })
+
+      .catch(error => console.log(error));
+  }
+});
+
+
+function imgCreate(data) {
+  if (data.results.length == 0) {
+    let p = document.createElement("p");
+    p.className = "picture";
+    p.innerHTML = "Таких картинок нет"
+    p.style.color = "white"
+    imgContainer.append(p);
+  }
```

Comment:
```suggestion
    const p = document.createElement("p");
    
    p.className = "picture";
    
    p.innerHTML = "Таких картинок нет";
    
    p.style.color = "white";
    
    imgContainer.append(p);
  }
```


1 - use ; in the end of the string
2 - use const instead of let
3 - use english please from now on "There is nothing to show you or there are no pictures with such name" etc.